packages/terra-clinical-data-grid/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-clinical-data-grid",
   "main": "lib/DataGrid.js",
-  "version": "2.35.1",
+  "version": "2.35.2",
```

Comment:
Doesn't this get updated during the release process?

Reply:
@sdadn Yes, I indicated in the PR description that this PR includes the version bump for publishing because I wanted to get this released ASAP and not wait until for week to unblock that team needing this. That way I don't have to issue a separate PR, wait for it build and get approvals again.